components/homme/src/share/cxx/utilities/InternalDiagnostics.hpp

```diff
@@ -0,0 +1,19 @@
+/********************************************************************************
+ * HOMMEXX 1.0: Copyright of Sandia Corporation
+ * This software is released under the BSD license
+ * See the file 'COPYRIGHT' in the HOMMEXX/src/share/cxx directory
+ *******************************************************************************/
+
+#ifndef INTERNAL_DIAGNOSTICS_HPP
+#define INTERNAL_DIAGNOSTICS_HPP
```

Comment:
Maybe here too, although it's probably less likely to get a macro clash than for `HASH_HPP` ...